css/style.css

```diff
@@ -95,6 +95,20 @@ div.recently-dropped-toggle-container {
 }
 
 /*
+ * ban list
+ */
+
+.ban-source-toggle-container {
+  text-align: right;
+  width: 116px;
+}
+
+.ban-source-toggle-container > ul {
+  text-align: left;
+  width: 150px;
+}
```

Comment:
I haven't pulled this branch down, but if the concern is locking in constraints, `rem` units might be the solution. The container will scale if the font changes.